spec/features/user_deletes_attachment_from_courses_page_spec.rb

```diff
@@ -27,4 +26,31 @@
       expect(page).to have_no_content(attachment_count)
     end
   end
+
+  scenario "related to a specific coverage successfully" do
```

Comment:
What are your thoughts on moving this out as its own spec? I think it would be good to keep this feature spec focused on courses only.